server/command.go

```diff
@@ -90,7 +95,7 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 
 	switch action {
 	case "subscribe":
-		features := "pulls,issues"
+		features := "pulls,issues,pushes,creates,deletes,issue_comments,pull_reviews"
```

Comment:
I wonder if we should default to less, do you think this might be too spammy by default?

Reply:
I would just set it to `"pulls,issues,creates,deletes". `issue_comments` and `pull_reviews` are way to spammy. `pushes` are debatable.